Makefile

```diff
@@ -157,7 +157,8 @@ diff_pylint_report: pylint_report.txt
 # statement). So we run nose inside of coverage.
 .coverage: $(PYSOURCES) $(wildcard tests/*.py) khmer/_khmermodule.so
 	coverage run --branch --source=scripts,khmer,oxli --omit=khmer/_version.py \
-		-m nose --with-xunit --attr=\!known_failing --processes=0
+		-m nose --with-xunit --attr=\!known_failing,\!huge \
```

Comment:
And use `TEST_ATTR ?= XXX` up the top of the makefile, and `export TEST_ATTR='!known_failing,!huge'` in jenkins.sh